ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -413,6 +413,30 @@ def test_source_ipa_roles_check_crlmanager(self):
             assert check["kw"]["key"] == "crl_manager"
             assert check["kw"]["crlgen_enabled"] is False
 
+    def test_ipa_healthcheck_log_rotate_file_exist_issue35(self):
+        """
+        This test checks if log rotation has been added
+        for ipa-healthcheck tool so that logs are rotated
+        in /var/log/ipa/healthcheck folder.
+        This is a testcase for below pagure issue
+        https://github.com/freeipa/freeipa-healthcheck/issues/35
+        Steps:
+        1. install ipa-healthcheck tool
+        2. Check logrotate file is present for healthcheck tool
+        3. Check contents of the ipahealthcheck file
+        """
+        tasks.install_packages(self.master, HEALTHCHECK_PKG)
+        if self.master.transport.file_exists(HEALTHCHECK_LOG_ROTATE_CONF):
+            contents = self.master.get_file_contents(
+                HEALTHCHECK_LOG_ROTATE_CONF, encoding="utf-8"
+            )
+            assert HEALTHCHECK_LOG in contents
+            assert "create" in contents
+            assert "daily" in contents
+            assert "dateext" in contents
+            assert "missingok" in contents
+            assert "rotate 30" in contents
+
```

Comment:
Instead of multiple assert statements, you can do like this:
check = ['create', 'daily', 'dateext', 'missingok', 'rotate 30']
for i in check:
   assert i in contents